store/src/sql/query.rs

```diff
@@ -151,6 +151,25 @@ impl EventQuery {
     pub fn mark_ready_to_deliver() -> &'static str {
         "UPDATE ceramic_one_event SET delivered = $1 WHERE cid = $2 and delivered is NULL;"
     }
+
+    /// Inserts a new high water mark after a batch has been anchored
+    /// Requires 1 parameter1:
+    ///     $1 = new_high_water_mark (i64)
+    pub fn insert_anchoring_high_water_mark() -> &'static str {
+        r#"INSERT INTO ceramic_one_anchor_high_water_mark (high_water_mark) VALUES ($1);"#
+    }
+
+    /// Fetch event CIDs from a specified source that are above the current anchoring high water mark
+    pub fn unanchored_from_high_water_mark() -> &'static str {
```

Comment:
how do we filter out time events? will source be different? it seems like it might be weird in the interim if js-ceramic is giving us time events and we're putting source == our address and then we anchor them again